addons/bus/models/bus.py

```diff
@@ -154,6 +157,23 @@ def _bus_last_id(self):
         return last.id if last else 0
 
 
+class BusSendableMixin(models.AbstractModel):
+    _name = 'bus.sendable'
+    _description = 'Can send messages via bus.bus'
+
+    @api.model
+    def _bus_send(self, notification_type, message):
+        if not self:
+            return
```

Comment:
I'm not sure if we want the quick return? We should never be in a situation where we think we send something to a user but there's no user.

Reply:
The quick return is managed by `for record in self` now, following discussion on the other (next) comment